.travis.yml

```diff
@@ -19,6 +19,7 @@ env:
   - TEST_TYPE=admin-e2e-3
   - TEST_TYPE=pupil-e2e-1
   - TEST_TYPE=pupil-e2e-2
+  - OVERRIDE_PIN_EXPIRY="true"
```

Comment:
this is not in the correct place.  it will cause a separate build with the other 5 types.  it needs to be in the global section to apply to the all parallel builds